src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Brushes.cs

```diff
@@ -247,9 +255,10 @@ public static void LoadVsBrushes(INuGetExperimentationService nuGetExperimentati
             ContentSelectedTextBrushKey = CommonDocumentColors.ListItemTextSelectedBrushKey;
 
             // Brushes/Colors for FilterLabel (Top Tabs)
-            TabSelectedBrushKey = CommonDocumentColors.InnerTabTextFocusedBrushKey;
-            TabSelectedTextBrushKey = CommonDocumentColors.InnerTabTextFocusedBrushKey;
-            TabHoverBrushKey = CommonDocumentColors.InnerTabInactiveHoverTextBrushKey;
+            TabSelectedBrushKey = CommonDocumentColors.InnerTabSelectedIndicatorBrushKey; // underline
+            TabSelectedTextBrushKey = CommonDocumentColors.InnerTabSelectedTextBrushKey; // text
+            TabHoverTextBrushKey = CommonDocumentColors.InnerTabTextHoverBrushKey; //text hover
```

Comment:
Since you're renaming some of these keys, anyway, why don't we match the same name from Common?

ex: 
```suggestion
            TabTextHoverBrushKey = CommonDocumentColors.InnerTabTextHoverBrushKey; //text hover
```